src/lib.rs

```diff
@@ -198,14 +218,68 @@ impl Expr {
         }
     }
 
-    pub fn as_json(&self) -> String {
-        serde_json::to_string(&self).unwrap()
+    /// Converts this expression to a JSON string.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use cql2::Expr;
+    ///
+    /// let expr = Expr::Bool(true);
+    /// let s = expr.as_json().unwrap();
+    /// ```
+    pub fn as_json(&self) -> Result<String, serde_json::Error> {
+        serde_json::to_string(&self)
+    }
+
+    /// Converts this expression to a pretty JSON string.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use cql2::Expr;
+    ///
+    /// let expr = Expr::Bool(true);
+    /// let s = expr.as_json_pretty().unwrap();
+    /// ```
+    pub fn as_json_pretty(&self) -> Result<String, serde_json::Error> {
```

Comment:
```suggestion
    /// let s = expr.to_json_pretty().unwrap();
    /// ```
    pub fn to_json_pretty(&self) -> Result<String, serde_json::Error> {
```